src/idl_gen_ts.cpp

```diff
@@ -214,6 +215,10 @@ class TsGenerator : public BaseGenerator {
 
   std::map<std::string, NsDefinition> ns_defs_;
 
+  // Value for non present optional fields. It's "null" by default and
```

Comment:
probably only the first sentence is needed. The code in the constructor documents the condition.